webui/src/App.js

```diff
@@ -145,22 +169,33 @@ function ValueMonitor(props) {
       }
     });
 
-    const render = () => {
-      // ********** Canvas setup **********
-      // This has to be here in case the user resizes the window.
+    const setDimensions = () => {
       // Adjust DPI so that all the edges are smooth during scaling.
       const dpi = window.devicePixelRatio || 1;
-      const style = {
-        height() {
-          return +getComputedStyle(canvas).getPropertyValue('height').slice(0,-2);
-        },
-        width() {
-          return +getComputedStyle(canvas).getPropertyValue('width').slice(0,-2);
-        }
+
+      canvas.width = canvas.clientWidth * dpi;
+      canvas.height = canvas.clientHeight * dpi;
+    };
+
+    setDimensions();
+    window.addEventListener('resize', setDimensions);
+
+    // Add background fill.
```

Comment:
This comment should be moved below to the createLinearGradient call

Reply:
done